nixos/modules/services/web-apps/plausible.nix

```diff
@@ -291,8 +291,23 @@ in {
                   '';
                 };
               in
+                # Note that setting `RELEASE_VM_ARGS` replaces the use of
+                # the `vm.args` file provided by the package.
+                # See: https://github.com/NixOS/nixpkgs/pull/130297#issuecomment-881128718
+                # As of writing, Plausible does not provide a `vm.args` file.
+                # In this script we also add an assertion about the absence
+                # of that file, so we notice when it changes.
                 pkgs.runCommand "plausible-vm-args" {} ''
                   set -eu -o pipefail
+
+                  # grep to print all lines that aren't comments, empty, or whitespace.
+                  FOUND_VM_ARG=0
+                  grep -v -E '^(#|$|[[:space:]]+)' "${pkgs.plausible}"/releases/*/vm.args || FOUND_VM_ARG=1
+                  if [ "$FOUND_VM_ARG" -eq 0 ]; then
```

Comment:
How about
```
(( FOUND_VM_ARG = 0 ))
```

Which to my knowledge is the recommended way to do arithmetic.
I don't have any strong feelings on this.

Reply:
That's right, however I am not doing arithmetic. Just normal exit code handling, for which it is common to use strings. For example `$?` also works that way.